### ISSUE TITLE
Translation error with Country-Kitchen Cycles scene

### ISSUE THREAD

**Issue by pablode** [GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon#208]
Tested with latest master (1.0.77) and Blender 3.0.1.

To reproduce:
1. open Country-Kitchen Cycles scene: https://www.blendswap.com/blend/5156
2. set Render Engine to 'USD Hydra'
3. activate 'Viewport Shading'
4. nothing is rendered; observe error in Blender console

```
2022-03-05 16:24:19,112 ERROR hdusd.engine [18612]:  'NoneType' object has no attribute 'nodes', EXCEPTION:, Traceback (most recent call last):
  File "C:\Users\pablode\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\hdusd\engine\engine.py", line 111, in view_update
    self.engine.sync(context, depsgraph)
  File "C:\Users\pablode\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\hdusd\engine\viewport_engine.py", line 200, in sync
    self._sync(context, depsgraph)
  File "C:\Users\pablode\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\hdusd\engine\viewport_engine.py", line 385, in _sync
    object.sync(root_prim, obj_data)
  File "C:\Users\pablode\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\hdusd\export\object.py", line 175, in sync
    mesh.sync(obj_prim, obj, **kwargs)
  File "C:\Users\pablode\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\hdusd\export\mesh.py", line 229, in sync
    _assign_materials(obj_prim, obj.original, usd_mesh)
  File "C:\Users\pablode\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\hdusd\export\mesh.py", line 235, in _assign_materials
    usd_mat = material.sync(obj_prim, obj.material_slots[0].material, obj)
  File "C:\Users\pablode\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\hdusd\export\material.py", line 41, in sync
    doc = mat.hdusd.export(obj)
  File "C:\Users\pablode\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\hdusd\properties\material.py", line 50, in export
    output_node = self.output_node
  File "C:\Users\pablode\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\hdusd\properties\material.py", line 41, in output_node
    return next((node for node in material.node_tree.nodes if
AttributeError: 'NoneType' object has no attribute 'nodes'
```

**Comment by DagerD** [comment]
Hi.

Thanks, confirmed as BLEN-58.

**Comment by DagerD** [comment]
Will be fixed with #214

**Comment by pablode** [comment]
Great, thanks!
